classes/02class/exercises/c02-network04/jayanath/README.md

```diff
@@ -0,0 +1,201 @@
+# c02-network04
+
+## Commands Execution Output
+
+- Commands for creating the subnets:
+```
+## Create the subnets
+    Subnet CIDRs
+    192.168.0.0/19
+    192.168.32.0/19
+    192.168.64.0/19
+    192.168.96.0/19
+    192.168.128.0/19
+    192.168.160.0/19
+    192.168.192.0/18 - not using this one
+## Keeping 192.168.192.0/18 range out to have /19 cidrs for all 6 subnets :-)
+
+## private-a -------------------------------------------------------------------------------------
+
+> aws ec2 create-subnet \
```

Comment:
Great use of the CLI Commands!
You'll understand the power of IaC in Class 4 to streamline all these !